### PR TITLE
Update ref_collectd-plugins.adoc

### PR DESCRIPTION
**Pull request by JoanneOFlynn2018**
commented out chrony
Updated connectivity, conntrack and contextswitch

### CODE REVIEW

File: common/collectd/ref_collectd-plugins.adoc [hunk 1]
```diff
@@ -241,27 +241,58 @@ Use the `cgroups` plugin to collect information for processes in a cgroup.
...
 
-[discrete]
-== collectd::plugin::chrony
+// To do: Add this to OSP 17 when it's available.
+//== collectd::plugin::chrony
 
 [discrete]
 == collectd::plugin::connectivity
+Use the connectivity plugin to monitor the state of network interfaces. 
+[NOTE]
```
Comment (leifmadsen) [inline common/collectd/ref_collectd-plugins.adoc:250]: Does this render properly? I think we need a newline between the NOTE and the line above it.

**Review by leifmadsen — APPROVED** [review]
Other than that one check for spacing, this looks good to me.

File: common/collectd/ref_collectd-plugins.adoc (hunk 1 above)
Comment (leifmadsen) [reply common/collectd/ref_collectd-plugins.adoc:250]: Ha! Not sure if this one failed, but the CI definitely caught some build issues :)

Comment (JoanneOFlynn2018) [reply common/collectd/ref_collectd-plugins.adoc:251]: Clever CI. I forgot to close the tables!

**Review by leifmadsen — APPROVED** [review]

**Pull request merged**